### PR TITLE
A couple of minor fixes

### PR DESCRIPTION
**Pull request by nealrichardson**
This is nice and clear, particularly in the context of the other chapters in the book. 

Would it make sense to link to the [main httptest vignette](https://enpiar.com/r/httptest/articles/httptest.html) somewhere? It goes deeper on some of these topics, and it describes an alternative workflow for writing tests (which admittedly is less intuitive than the one you present here, but it shows some other useful testing functionality).

### CODE REVIEW

**Comment by maelle** [comment]
Thank you! Maybe a sentence at the end of the chapter could introduce this vignette and link to it?

**Comment by nealrichardson** [comment]
Yeah, that could work. As the package author, the whole-game felt a little strange to read since it only uses a single, unreleased function/wrapper, so it would be nice to suggest that there's lots more you can do with `httptest`.

**Comment by maelle** [comment]
Or one of the tests could use another approach if you think it'd be a better demo?

**Comment by maelle** [comment]
I am a bit torn because I want the demo to remain simple 🤔 in vcr demo I mention another approach (to error testing) in a blue box, maybe I could do the same, and then re-mention the vignette in the conclusion?

**Comment by nealrichardson** [comment]
Yeah, I agree that it's nice that the demo is simple. It's just odd because none of the 43 packages on CRAN that use httptest use the method in the demo. Some sort of callout like you suggest sounds good.

**Comment by maelle** [comment]
I'll make a proposal tomorrow!

**Comment by maelle** [comment]
Is the added box ok? Note that https://books.ropensci.org/http-testing/conclusion-4.html now also has an item about reverse dependencies.

I see downlit generates a wrong link, to https://enpiar.com/r/httptest/reference/with_mock_api.html -- is there a way to fix this on httptest's side (some sort of roxygen2 tweak) or should I fix it here?

**Comment by maelle** [comment]
If you have time to read the two comparison chapters, your thoughts on them would be welcome too (well one of them has a whole quote of yours, that part can't be wrong :grin:)

* httptest and vcr, as both use mocking https://books.ropensci.org/http-testing/mocking-pkgs-comparison.html
* general comparison https://books.ropensci.org/http-testing/pkgs-comparison.html

**Comment by maelle** [comment]
Once there are docs for https://github.com/nealrichardson/httptest/issues/43 I'll also link to that from the box.

File: wholegames-httptest.Rmd [hunk 1]
```diff
@@ -110,6 +110,12 @@ Without the HTTP testing infrastructure, testing for behavior of the package in
 
 Regarding our secret API token, since httptest doesn't save the requests, and since the responses don't contain the token, it is safe without our making any effort.
 
+::: {.alert .alert-dismissible .alert-primary}
+In this demo we used `httptest::with_mock_dir()` but there are other ways to use httptest, e.g. using `httptest::with_mock_api()` that does not require naming a directory (you'd still need to use a separate directory for mocking the error response).
+ 
+Find our more in the [main httptest vignette](https://enpiar.com/r/httptest/articles/httptest.html), especially if you don't find the `httptest::with_mock_dir()` approach enjoyable.
```
Comment (nealrichardson) [inline wholegames-httptest.Rmd:116]:
```suggestion
Find out more in the [main httptest vignette](https://enpiar.com/r/httptest/articles/httptest.html).
```

**Comment by nealrichardson** [comment]
One wording suggestion but that looks good to me, thanks! I read the other chapters earlier in the week and thought they looked good.

I'll fix the with_mock_api.html link again, it's the same issue I fixed before with the macOS case-insensitive file system, need to fix it on GitHub, delete my local repo, and make a fresh clone.

**Comment by maelle** [comment]
Thanks!

**Pull request merged**